.golangci.yml

```diff
@@ -27,10 +27,11 @@ linters-settings:
       - ifElseChain # Mostly false positives
       - ptrToRefParam # False positives?
       - importShadow # Probably not worth the hassle...
-  depguard:
-    include-go-root: true
-    packages:
-      - "io/ioutil"
+  # depguard:
```

Comment:
Leaving this here as an example. We can re-enable this linter (it's commented out below) when we'll need it.